docs/en/00_Getting_Started/02_Composer.md

```diff
@@ -56,48 +64,57 @@ see [Using development versions](#using-development-versions).
 
 Composer isn't only used to download SilverStripe CMS, it can also be used to manage all SilverStripe modules.  Installing a module can be done with the following command:
 
-	composer require "silverstripe/forum:*"
+```
+composer require "silverstripe/forum:*"
+```
+
+This will install the forum module in the latest compatible version. You can also omit the `:*` if you simply want the latest compatible version of the module.
```

Comment:
IMHO using "*" as a version is a bad practice. If all modules are using proper semver you should use ^1.2.3 to update in the v1 branch automatically (without updating your composer json) but use minimal v1.2.3, or 1.2.x to use the latest tagged version of v1.2, but not update to 2.

@stevie-mayhew From my experience composer will install the latest compatible version or throw an error, even with minimum-stability:dev, it will only result in incompatible code if the module's composer.json doesn't have the right requirements to the framework etc.